pkg/doc/jsonld/contexts/bbs2020.jsonld

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
TODO: might want to more clearly indicate that these are embed data in the directory structure.